### ISSUE TITLE
Netatmo tags have no / wrong values

### ISSUE THREAD

**Issue by hnijveen** [athombv/homey#1312]
**What is your Homey version (Settings → System)?**

Homey Firmware Version: 1.1.4

**What did you try to achieve?**

Switch on / off zwave wall plug using Netatmo Humidity value tags

*Please include detailed steps how we can reproduce the issue*

Check global tags or tags in a simple flow:

<img width="1348" alt="screen shot 2017-02-05 at 09 34 46" src="https://cloud.githubusercontent.com/assets/16666747/22624811/e688f400-eb86-11e6-80a0-9f09a99526d5.png">

**What was the result?**
tags are empty
...

**What did you expect as result?**

when tag is below certain value, condition is met and flow will be executed
...

**Any other remarks**

...

**Comment by bvdbos** [comment]
Better put it here:
https://github.com/athombv/com.netatmo

**Comment by Dijker** [comment]
Hi @hnijveen , 
I have Checked this here on both my Homey's  1.1.2 and 1.1.4 and on NetAtmo App 2.0.5 as I want to update my production Homey to 1.1.4.

My tags for the NetAtmo Weather station work normal on both Homey's. 

did it work before? and is this only the taggs not filling? 
do the NetAtmo device and Insights report correct values?
Did you already look if the NetAtmo app isn't crashed? What version has your App?

**Comment by hnijveen** [comment]
Hi all, sorry I was not in a possition to answer at an earlier state. I work abroad (not where my Homey is) and only check & configure Homey in the weekend. After I listed the github issue, I also decided to delete the Netatmo device in Homey and add it again. Re-enbled the flows, and today I check with my wife how did Homey work for you? Reply: Great, the humidity machine acted as I promised to her (ehh Homey). So all is ok now, after a week of testing. One tip: get a faq, what to do when something does not work. Add remove and re-add device as a standard knowing you are so busy working on the basic fundament of the Homey-OS. Thanks, guys (girls) and close this issue.
